plugins/node/opentelemetry-instrumentation-hapi/src/utils.ts

```diff
@@ -94,6 +94,20 @@ export const getRouteMetadata = (
   };
 };
 
+export const getRootSpanMetadata = (
+  route: Hapi.ServerRoute
+): {
+  attributes: SpanAttributes;
+  name: string;
+} => {
+  return {
+    attributes: {
+      [SemanticAttributes.HTTP_ROUTE]: route.path,
+    },
+    name: `${route.method} ${route.path}`,
```

Comment:
Is `route.path` the parameterized path like `/users/:id` or the full path like `/users/123`?